### PR TITLE
Ranjaniguru/#931 Error while installing data catalog

### LINKED ISSUE #931: Error while installing data catalog, when generating RDBMS shape

### PR DESCRIPTION
**Pull request by RanjaniGuru**
Fixes #931

### CODE REVIEW

File: konig-data-catalog/src/main/java/io/konig/datacatalog/PropertyPage.java [hunk 1]
```diff
@@ -91,8 +92,9 @@ private void setEquivalentPath(PropertyRequest request) throws DataCatalogExcept
 			
 			List<PathElementView> list = new ArrayList<>();
 			for (Step step : path.asList()) {
-				
-				list.add(pathElement(request, step));
+				if(!(step instanceof InStep)){
```
Comment (gmcfall) [inline konig-data-catalog/src/main/java/io/konig/datacatalog/PropertyPage.java]: We should not skip InStep elements as shown here.  Instead, the `pathElement` method should handle the InStep element gracefully (and not throw an exception).  An InStep element should be handled in the same way as an OutStep element but with the caret ("^") as the operator.

File: konig-schemagen/src/main/java/io/konig/schemagen/sql/RdbmsShapeGenerator.java [hunk 2]
```diff
@@ -138,21 +137,20 @@ private void process(Shape rdbmsShape, String logicalPath, String prefix, Shape
 						
 					URI newPredicate =  new URIImpl( propertyNameSpace + newLocalName );
 					
-					declarePredicate(newPredicate);
+					declarePredicate(predicate);
```
Comment (gmcfall) [inline konig-schemagen/src/main/java/io/konig/schemagen/sql/RdbmsShapeGenerator.java]: Why did you change this???

We need to declare the newPredicate.  The original predicate should already be declared.

**Comment by RanjaniGuru** [comment]
Code updated as per the comments mentioned.
